File: languages/fortran/basics/01_hello.f90
! Objetivo: compilar e executar o primeiro programa em Fortran.
! Como executar:
!   powershell -ExecutionPolicy Bypass -File .\scriptsuild-fortran-module.ps1 -Source .\languagesortranasics_hello.f90
! Conceitos treinados: program, implicit none, print.
! Próximo exercício: 02_variaveis_tipos.f90

program hello_fortran
    implicit none
    print *, "Ola, Fortran moderno no VS Code!"
end program hello_fortran

File: languages/cpp/basics/01_hello.cpp
/*
Objetivo: compilar e executar o primeiro programa em C++.
Como executar:
  powershell -ExecutionPolicy Bypass -File .\scripts\build-cpp-module.ps1 -Source .\languages\cpp\basics\01_hello.cpp
Conceitos treinados: iostream, main.
Próximo exercício: 02_variaveis_tipos.cpp
*/

#include <iostream>

int main() {
    std::cout << "Olá, C++ no VS Code!" << std::endl;
    return 0;
}
